modules/core/shared/src/main/scala/gem/config/DynamicConfig.scala

```diff
@@ -123,6 +164,11 @@ object DynamicConfig {
       Step.Niri(this, b)
```

Comment:
Why `case class Niri()` rather than `case object Niri`?

Reply:
This was done some time ago.  It does make all the instrument cases the same at least.  Eventually they'll all need to be filled out with configuration details like `GmosN`, etc.  If some of the instrument configs are objects and some are classes then it becomes awkward like `Step.Bias` and `Step.Dark`.  For example, the steps need a `DynamicConfig` instance and so it'd have to be something like `sealed case class Niri(dynamicConfig: DynamicConfig.Niri.type, base: Base) extends Step` until eventually we add the actual config fields and have to switch it over.

There may have been other reasons.

Reply:
Making it, say,

```
sealed case class Niri(base: Base) extends Step {
  override def dynamicConfig: DynamicConfig =
    Niri
}
```
would be even more fiddling later when `Niri` is ultimately defined.

Reply:
👍